client/app/(todos)/hooks/useUpdateStatusMutation.ts

```diff
@@ -0,0 +1,22 @@
+import { useMutation } from '@tanstack/react-query'
+import { useQueryClient } from '@tanstack/react-query'
+import { useAuthStore } from '@/_stores'
+import { updateTaskStatusApi } from './api'
+
+export const useUpdateStatusMutation = () => {
+  const queryClient = useQueryClient()
+  const getHouseId = useAuthStore((state) => state.getHouseId)
+  const houseId = getHouseId()
+
+  const { mutate } = useMutation({
+    mutationFn: updateTaskStatusApi,
+    onSuccess: () => {
+      queryClient.invalidateQueries({ queryKey: ['todos', houseId] })
+    },
+    onError: (error) => {
+      console.error('Update failed', error)
+    },
```

Comment:
Show toast? We need to handle mutation errors (This is different from query handling, I will do query handling later)

Reply:
I added toast! I'm not sure about mutation errors you're talking about, so let me follow it when you do it!
https://github.com/HAK2024/fairy-share/pull/71/commits/3160c1d542a54421905181f9b034656dc1fcc526